WcaOnRails/app/views/results_submission/edit.html.erb

```diff
@@ -0,0 +1,15 @@
+<% provide(:title, "Results submission") %>
+<%= render layout: 'nav' do %>
+  <h1><%= yield(:title) %></h1>
+
+  <p>Enter the body of your email to the Results Team.</p>
+  <p>Make sure to include any changes to the original schedule, as well as any other additional details required by the 'Submitting competition results' section of the <%= link_to "delegate crash course", delegate_crash_course_path %> .</p>
```

Comment:
We should link directly to the relevant section of the delegate crash course here instead:

I think this bit of code will generate the correct link (https://www.worldcubeassociation.org/delegate/crash-course#submitting-competition-results)
```ruby
delegate_crash_course_path(anchor: "submitting-competition-results")
```

Reply:
Changed!
I don't have a local delegate crash course locally but it does redirect to http://localhost:3000/delegate/crash-course#submitting-competition-results